eng/pipelines/linux.yml

```diff
@@ -105,11 +114,13 @@ jobs:
           - linuxArm64Queues: \(Ubuntu.1604.Arm64\)Ubuntu.1604.Arm64.Docker.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm64v8-b049512-20190321153539
           - alpineQueues: \(Alpine.38.Amd64\)ubuntu.1604.amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.8-helix-45b1fa2-20190327215821
       
+          - alpineArm64Queues: \(Alpine.38.Arm64\)Ubuntu.1604.Arm64.Docker.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.8-helix-arm64v8-46e69dd-20190327215724
```

Comment:
Nit: remove extra line before and add an extra line after.

Reply:
Actually, this is not needed since we're not running alpine arm tests in CI yet. So remove this line

Reply:
I left it there on purpose as the time will come. And I always miss something (like adding .Open) 
But that can be done in future so I'll remove it if that is preferred.

Reply:
Thanks 😄